.github/workflows/run-tests.yml

```diff
@@ -25,4 +25,24 @@ jobs:
       - run: npm run build
       - run: npm run test
         env:
-          FORCE_COLOR: 1
\ No newline at end of file
+          FORCE_COLOR: 1
+      - name: Notify about failures
+        if: failure()
+        uses: slackapi/slack-github-action@v1
+        with:
+          payload: |
+            {
+              "text": "",
```

Comment:
Testing revealed that the `text` root-level property does not get included in the printed Slack message.